pkgs/data/fonts/googlefonts/default.nix

```
@@ -0,0 +1,200 @@
+{ stdenvNoCC
+, lib
+, fetchFromGitHub
+, symlinkJoin
+}: let
+  version = "unstable+20230722";
```

This would be idiomatic:
```suggestion
  version = "unstable-2023-07-22";
```
Should probably also be `git commit --amend`'ed in the commit summary.